apps/mobile/src/stacks/Dashboard/TabBar.tsx

```diff
@@ -59,36 +47,60 @@ export const BottomNavigationTabBar: FC<Props> = ({ tabProps }) => {
 		offset.value = withTiming(nextOffset, timingConfig);
 	}, [isDrawerOpen]);
 
+	useEffect(() => {
+		if (appState.profile.profileImage) {
+			ChildrenList[2] = (
+				<Image
+					key="Setting"
+					style={{
+						width: iconSize,
+						height: iconSize,
+						borderRadius: iconSize / 2,
+					}}
+					source={{ uri: appState.profile.profileImage }}
+				/>
+			);
+		}
+	}, []);
```

Comment:
don't do this, `1.` assign component via hook is bad, `2.` the hook itself run once, which we'll stuck with the initial-rendering even if `appSate.profile` changed.